tools/Azure.Mcp.Tools.Monitor/tests/Azure.Mcp.Tools.Monitor.LiveTests/MonitorCommandTests.cs

```diff
@@ -452,5 +456,158 @@ private async Task GenerateStorageActivityAsync()
             // Don't fail the test if storage activity generation fails
         }
     }
-}
 
+    #region WebTests Integration Tests
+
+    [Fact]
+    public async Task Should_List_WebTests()
+    {
+        var result = await CallToolAsync(
+            "azmcp_monitor_webtests_list",
+            new()
+            {
+                { "subscription", Settings.SubscriptionId }
+            });
+
+        var webTestsArray = result.AssertProperty("webTests");
+        Assert.Equal(JsonValueKind.Array, webTestsArray.ValueKind);
+
+        // Should have at least 2 web tests (bing and microsoft from test resources)
+        var webTests = webTestsArray.EnumerateArray().ToList();
+        Assert.True(webTests.Count >= 2, $"Expected at least 2 web tests, but found {webTests.Count}");
+
+        foreach (var webTest in webTests)
+        {
+            Assert.True(webTest.TryGetProperty("resourceName", out var resourceName));
```

Comment:
Use the `JsonElement.AssertProperty` to validate required properties, `TryGetProperty` returns less useful debugging data.

```suggestion
            var resourceName = webTest.AssertProperty("resourceName");
```